frontend/public/components/storage/create-pvc.tsx

```diff
@@ -117,6 +189,16 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
   ]);
 
   const { t } = useTranslation();
+
+  const InfoToolTip = () => (
```

Comment:
I might be missing it, but where is this used?

It would be better to make this its own component defined outside of this one.

Reply:
Looks like caller was removed when I rebased. I'll take this out.